{"uid":"1c193bebbc66c429","name":"test_accept_order_with_invalid_courier_id","fullName":"tests.test_return_order.TestCreatingCourier#test_accept_order_with_invalid_courier_id","historyId":"4019f7e6b1ec1de3be5d8fa231d5dceb","time":{"start":1760439188598,"stop":1760439189007,"duration":409},"status":"failed","statusMessage":"AssertionError: Ожидался ответ {\"message\": \"Курьера с таким id не существует\"}, но получили {'code': 404, 'message': 'Курьера с таким id не существует'}\nassert {'code': 404,...е существует'} == {'message': '...е существует'}\n  \n  Omitting 1 identical items, use -vv to show\n  Left contains 1 more item:\n  \u001B[0m{\u001B[33m'\u001B[39;49;00m\u001B[33mcode\u001B[39;49;00m\u001B[33m'\u001B[39;49;00m: \u001B[94m404\u001B[39;49;00m}\u001B[90m\u001B[39;49;00m\n  \n  Full diff:\n  \u001B[0m\u001B[90m \u001B[39;49;00m {\u001B[90m\u001B[39;49;00m...\n  \n  ...Full output truncated (3 lines hidden), use '-vv' to show","statusTrace":"self = <test_return_order.TestCreatingCourier object at 0x000001D650E11810>, prepared_order = (561093, 328018)\n\n    def test_accept_order_with_invalid_courier_id(self, prepared_order):\n        with allure.step(\"Принять заказ с неверным id курьера\"):\n            order_id, _ = prepared_order\n            resp = requests.put(f'{Url.url}{Url.put_order}/{order_id}?courierId=999999')  # Неверный ID курьера\n    \n            assert resp.status_code == 404, f\"Ожидался статус 404, но получили {resp.status_code}\"\n>           assert resp.json() == {\"message\": \"Курьера с таким id не существует\"}, \\\n            f'Ожидался ответ {{\"message\": \"Курьера с таким id не существует\"}}, но получили {resp.json()}'\nE           AssertionError: Ожидался ответ {\"message\": \"Курьера с таким id не существует\"}, но получили {'code': 404, 'message': 'Курьера с таким id не существует'}\nE           assert {'code': 404,...е существует'} == {'message': '...е существует'}\nE             \nE             Omitting 1 identical items, use -vv to show\nE             Left contains 1 more item:\nE             \u001B[0m{\u001B[33m'\u001B[39;49;00m\u001B[33mcode\u001B[39;49;00m\u001B[33m'\u001B[39;49;00m: \u001B[94m404\u001B[39;49;00m}\u001B[90m\u001B[39;49;00m\nE             \nE             Full diff:\nE             \u001B[0m\u001B[90m \u001B[39;49;00m {\u001B[90m\u001B[39;49;00m...\nE             \nE             ...Full output truncated (3 lines hidden), use '-vv' to show\n\ntests\\test_return_order.py:31: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1760438909140,"stop":1760438909179,"duration":39},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0},{"name":"prepared_order","time":{"start":1760439187734,"stop":1760439188597,"duration":863},"status":"passed","steps":[{"name":"Создаем тестовый заказ","time":{"start":1760439187734,"stop":1760439188594,"duration":860},"status":"passed","steps":[],"attachments":[],"parameters":[],"hasContent":false,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":false,"attachmentStep":false,"attachmentsCount":0,"stepsCount":1}],"testStage":{"status":"failed","statusMessage":"AssertionError: Ожидался ответ {\"message\": \"Курьера с таким id не существует\"}, но получили {'code': 404, 'message': 'Курьера с таким id не существует'}\nassert {'code': 404,...е существует'} == {'message': '...е существует'}\n  \n  Omitting 1 identical items, use -vv to show\n  Left contains 1 more item:\n  \u001B[0m{\u001B[33m'\u001B[39;49;00m\u001B[33mcode\u001B[39;49;00m\u001B[33m'\u001B[39;49;00m: \u001B[94m404\u001B[39;49;00m}\u001B[90m\u001B[39;49;00m\n  \n  Full diff:\n  \u001B[0m\u001B[90m \u001B[39;49;00m {\u001B[90m\u001B[39;49;00m...\n  \n  ...Full output truncated (3 lines hidden), use '-vv' to show","statusTrace":"self = <test_return_order.TestCreatingCourier object at 0x000001D650E11810>, prepared_order = (561093, 328018)\n\n    def test_accept_order_with_invalid_courier_id(self, prepared_order):\n        with allure.step(\"Принять заказ с неверным id курьера\"):\n            order_id, _ = prepared_order\n            resp = requests.put(f'{Url.url}{Url.put_order}/{order_id}?courierId=999999')  # Неверный ID курьера\n    \n            assert resp.status_code == 404, f\"Ожидался статус 404, но получили {resp.status_code}\"\n>           assert resp.json() == {\"message\": \"Курьера с таким id не существует\"}, \\\n            f'Ожидался ответ {{\"message\": \"Курьера с таким id не существует\"}}, но получили {resp.json()}'\nE           AssertionError: Ожидался ответ {\"message\": \"Курьера с таким id не существует\"}, но получили {'code': 404, 'message': 'Курьера с таким id не существует'}\nE           assert {'code': 404,...е существует'} == {'message': '...е существует'}\nE             \nE             Omitting 1 identical items, use -vv to show\nE             Left contains 1 more item:\nE             \u001B[0m{\u001B[33m'\u001B[39;49;00m\u001B[33mcode\u001B[39;49;00m\u001B[33m'\u001B[39;49;00m: \u001B[94m404\u001B[39;49;00m}\u001B[90m\u001B[39;49;00m\nE             \nE             Full diff:\nE             \u001B[0m\u001B[90m \u001B[39;49;00m {\u001B[90m\u001B[39;49;00m...\nE             \nE             ...Full output truncated (3 lines hidden), use '-vv' to show\n\ntests\\test_return_order.py:31: AssertionError","steps":[{"name":"Принять заказ с неверным id курьера","time":{"start":1760439188598,"stop":1760439189005,"duration":407},"status":"failed","statusMessage":"AssertionError: Ожидался ответ {\"message\": \"Курьера с таким id не существует\"}, но получили {'code': 404, 'message': 'Курьера с таким id не существует'}\nassert {'code': 404,...е существует'} == {'message': '...е существует'}\n  \n  Omitting 1 identical items, use -vv to show\n  Left contains 1 more item:\n  \u001B[0m{\u001B[33m'\u001B[39;49;00m\u001B[33mcode\u001B[39;49;00m\u001B[33m'\u001B[39;49;00m: \u001B[94m404\u001B[39;49;00m}\u001B[90m\u001B[39;49;00m\n  \n  Full diff:\n  \u001B[0m\u001B[90m \u001B[39;49;00m {\u001B[90m\u001B[39;49;00m...\n  \n  ...Full output truncated (3 lines hidden), use '-vv' to show\n","statusTrace":"  File \"C:\\Users\\ilua2\\project\\qa_Sprint_7\\tests\\test_return_order.py\", line 31, in test_accept_order_with_invalid_courier_id\n    assert resp.json() == {\"message\": \"Курьера с таким id не существует\"}, \\\n    f'Ожидался ответ {{\"message\": \"Курьера с таким id не существует\"}}, но получили {resp.json()}'\n","steps":[],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentStep":false,"attachmentsCount":0,"stepsCount":0}],"attachments":[],"parameters":[],"hasContent":true,"shouldDisplayMessage":true,"attachmentStep":false,"attachmentsCount":0,"stepsCount":1},"afterStages":[],"labels":[{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_return_order"},{"name":"subSuite","value":"TestCreatingCourier"},{"name":"host","value":"Ilya_Volkov"},{"name":"thread","value":"5776-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_return_order"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[]}],"tags":[]},"source":"1c193bebbc66c429.json","parameterValues":[]}